package.json

```diff
@@ -36,8 +36,6 @@
     "ember-cli-babel": "^7.26.11",
     "ember-cli-htmlbars": "~6.2.0",
     "ember-cli-sass": "~11.0.1",
-    "ember-in-element-polyfill": "~1.0.1",
-    "ember-maybe-in-element": "2.0.3",
```

Comment:
`ember-maybe-in-element` ignores the fact that polyfill is added to the app's dependency list and throws an error in ember-source `< 3.20`:

`Assertion Failed: The {{in-element}} helper cannot be used.`

I think we can provide the custom component to replace that addon.